data/json/items/comestibles.json

```diff
@@ -825,6 +825,7 @@
         "volume" : 1,
         "phase" : "liquid",
         "charges" : 5,
+        "flags": [ "EATEN_COLD" ]
```

Comment:
Every one of your line additions need commas on the ends. The only line of a json array that doesn't end with a comma is the very final entry.